autotools.sh

```diff
@@ -0,0 +1,14 @@
+curl -OL http://ftpmirror.gnu.org/autoconf/autoconf-2.69.tar.gz
```

Comment:
This file could us a brief comment describing what it does.

Also #!/bin/bash at the top